### PR TITLE
Workaround to fix an issue with DS 3.8.16.3566.

### PR DESCRIPTION
**Pull request by p0psicles**
- [x] PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review
- [x] Read the [contribution guide](https://github.com/pymedusa/Medusa/blob/master/.github/CONTRIBUTING.md)

Fix issue #9366

### CODE REVIEW

File: medusa/clients/torrent/downloadstation.py [hunk 1]
```diff
@@ -126,6 +126,10 @@ def _add_torrent_uri(self, result):
         return self._check_response()
 
     def _add_torrent_file(self, result):
+        # The API in the latest version of Download Station (3.8.16.-3566)
+        # is broken for downloading via a file, only uri's are working correct.
+        if result.url[:4].lower() in ['http', 'magn']:
```
Comment (medariox) [inline medusa/clients/torrent/downloadstation.py:131]: What about .startswith()?

Comment (BenjV) [reply medusa/clients/torrent/downloadstation.py:131]: startswith() make is more complex, you have to do two test one for magnet and on for http.
This code test for http, https, and magnet links.

Comment (medariox) [reply medusa/clients/torrent/downloadstation.py:131]: You can use a tuple like: .startswith(('http', 'magn'))

Comment (BenjV) [reply medusa/clients/torrent/downloadstation.py:131]: There are many ways to Rome mine is tested

Comment (p0psicles) [reply medusa/clients/torrent/downloadstation.py:131]: yeah your both right. But i'll go with as is. Tnx for the contribution @benjv

**Pull request merged**